lib/PreviewProvider.js

```diff
@@ -82,7 +82,7 @@ PreviewProvider.prototype = {
       // filter out the allowed query params and update the query string
       newURL.search = Array.from(queryParams.keys())
         .filter(param => ALLOWED_QUERY_PARAMS.has(param))
-        .map(param => param = param + "=" + queryParams.get(param)).join("&");
+        .map(param => param + "=" + queryParams.get(param)).join("&");
```

Comment:
Seems like another logic change, but the old behavior looks decidedly odd with the extra `param +`.

Nit: Not sure if you want to move that `.join("&")` onto the following line so it aligns with `.filter()` and `.map()`.

Also not sure if we need to URL-escape the `queryParams.get(param)` value, or if that is done elsewhere.